azavea/echo-locator
# Feature/kak/extra markers#97

## Overview

Display differently styled map markers for neighborhoods listed in the sidebar (#97).

Also disables dragging end markers and adds back style linting to `yarn test`.


### Demo

![image](https://user-images.githubusercontent.com/960264/55416732-5ccfd080-553d-11e9-87fe-3e6c5032f26c.png)


### Notes

This required refactoring to move the pagination logic out of the sidebar state to share with the map.


## Testing Instructions

 * Go to map page
 * Should have blue markers with no inset icon for non-active neighborhoods listed in sidebar
 * Clicking a blue marker should go to the details view for that neighborhood
 * Pagination should update markers
 * Toggling between saved and full list and between details and list should still work as expected


Closes #90 
Closes #97 
Closes #69
